Gemfile

```diff
@@ -48,7 +48,8 @@ group :development, :test, :replica do
   gem 'minitest-rails'
   gem 'minitest-hyper'
   gem 'database_cleaner'
-  gem "minitest-osx"
+  gem 'minitest-osx'
+  gem 'minitest-around'
```

Comment:
@jakerenzella Did we actually end up using `minitest-around`? I remember we were trying it out for something but that didn't work... do we still need it?